gradle/libs.versions.toml

```diff
@@ -49,7 +49,7 @@ reactor-core = "3.4.14"
 reactor-netty = "1.0.15"
 scala-library = "2.13.9"
 servlet-api = "2.5"
-snakeyaml = "2.2"
+snakeyaml = "1.33"
```

Comment:
Looks like this version is still affected
https://nvd.nist.gov/vuln/detail/CVE-2022-1471

Reply:
Yeah, this might not be avoidable unless we upgrade to version `2.2` which causes a lot of issues.

I will investigate the impact...

Reply:
snakeyaml `2.2` breaks Spring Boot `2.7.x`. Upgrading to Spring `3.2.x` will resolve the snakeyaml vulnerability.

Reply:
I think we are not very affected by snakeyaml so let's fix it in the upcoming versions